core/templates/pages/header_js_libs.html

```diff
@@ -1,7 +1,7 @@
 <script src="/third_party/static/jquery-3.5.1/jquery.min.js"></script>
 <script src="/third_party/static/jqueryui-1.12.1/jquery-ui.min.js"></script>
-<script async src="https://www.googletagmanager.com/gtag/js"></script>
-<script async src="https://www.googleoptimize.com/optimize.js?id=OPT-57KK7PG"></script>
+<script async src="//www.googletagmanager.com/gtag/js"></script>
+<script async src="//www.googleoptimize.com/optimize.js?id=OPT-57KK7PG"></script>
```

Comment:
Why is 'https:' removed here?

Reply:
Hm, this means it just uses the protocol that the main Oppia page uses, which is https: https://stackoverflow.com/questions/9161769/url-without-httphttps

But I guess it doesn't hurt to keep it explicit. I'll do that.